src/views/PopupView.tsx

```diff
@@ -0,0 +1,62 @@
+import * as React from "react";
+import * as cameraControlsImg from "../../assets/images/camera-controls.png";
+import * as selectionControlsImg from "../../assets/images/selection-controls.png";
+
+interface Props {
+    error: Error|null,
+    menu: boolean,
+    sdpInfo: boolean,
+
+    controlsHelpImgNum: number,
+    setControlsHelpImgNum: React.Dispatch<React.SetStateAction<number>>,
+}
+
+export default function PopupView(
+    {
+        error, menu, sdpInfo, controlsHelpImgNum, setControlsHelpImgNum
+    }: Props
+) {
+    return <div id={"popup-view"}>
+
+        <div id={"error-popup"} style={error ? {visibility: 'visible'} : {visibility: 'hidden'}}>
```

Comment:
combining all these popups isn't very nice

Reply:
With my very limited frontend experience, I don't really know where to begin with a different approach. I tried for quite a while to get conditional rendering to work as you recommended to Connor (which I discussed in some detail in #165). If I had more time, maybe I could figure something out, but the tradeoff (that I'm not sure is worth making) would be to sacrifice time that could be spent delivering more of the features the sponsor requested. For the simple popups, I don't think the performance costs of this approach are too bad. If you want though, you're welcome to jump in and change these.